tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Differ;
+
+use PhpCsFixer\Differ\SebastianBergmannDiffer;
+use PHPUnit\Framework;
+use SebastianBergmann\Diff\Differ;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Differ\SebastianBergmannDiffer
+ */
+final class SebastianBergmannDifferTest extends Framework\TestCase
+{
+    public function testIsDiffer()
+    {
+        $differ = new SebastianBergmannDiffer();
+
+        $this->assertInstanceOf('PhpCsFixer\Differ\DifferInterface', $differ);
+    }
+
+    public function testDiffReturnsDiff()
+    {
+        $old = 'foo';
+        $new = 'bar';
+        $diff = 'baz';
+
+        $composedDiffer = $this->createDifferMock();
+
+        $composedDiffer
+            ->expects($this->once())
+            ->method('diff')
+            ->with(
+                $this->identicalTo($old),
+                $this->identicalTo($new)
+            )
+            ->willReturn($diff);
+
+        $differ = new SebastianBergmannDiffer($composedDiffer);
+
+        $this->assertSame($diff, $differ->diff($old, $new));
+    }
+
+    /**
+     * @return \PHPUnit_Framework_MockObject_MockObject|Differ
+     */
+    private function createDifferMock()
+    {
+        return $this->getMock('SebastianBergmann\Diff\Differ');
```

Comment:
btw, across whole project we are not using old phpunit3-style mocking, but prophecy (which was integrated in 4.5 already and is described as best practice)